include/vast/ABI/Classify.hpp

```diff
@@ -756,8 +571,7 @@ namespace vast::abi {
                 }
                 case Class::SSE: {
                     ++needed_sse;
-                    auto target_type =
-                        TypeConfig::sse_target_type_at_offset(t, 8, t, 8, mk_classifier_ctx());
+                    auto target_type = sse_target_type_at_offset(t, 64, t, 64);
```

Comment:
8 changed to 64. Was it intended?

Reply:
Yes, before it was bug because we were sending in size in bytes (as clang does) but our helpers return sizes in bits. So I unified it and used bits all over the place. This will really benefit from having strong types for bit/byte sizes, there is already an open issue for that.